src/components/shareable-url-modal/shareable-url-metadata.js

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from 'react';
+import { isRunningLocally } from '../../utils';
+
+const ShareableUrlMetadata = () => {
+  const [metadata, setMetadata] = useState(null);
+
+  useEffect(() => {
+    if (isRunningLocally()) {
+      return;
+    }
+
+    async function fetchData() {
+      try {
+        const request = await fetch('/api/deploy-viz-metadata', {
+          headers: {
+            'Content-Type': 'application/json',
+            Accept: 'application/json',
+          },
+        });
+        const response = await request.json();
+
+        if (request.ok) {
+          setMetadata(response);
+        }
+      } catch (error) {
+        console.log('deploy-viz-metadata fetch error: ', error);
```

Comment:
We should think of a better way to display an error or at least replace the `log` with `warn` or `error`.